public/app/core/components/TransformersUI/FilterByValueTransformerEditor.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useMemo } from 'react';
+import {
+  DataTransformerID,
+  standardTransformers,
+  TransformerRegistyItem,
+  TransformerUIProps,
+  getFieldDisplayName,
+  Field,
+  DataFrame,
+} from '@grafana/data';
+import { getAllFieldNamesFromDataFrames } from './OrganizeFieldsTransformerEditor';
+import { Select, Button, Input } from '@grafana/ui';
+
+import {
+  FilterByValueTransformerOptions,
+  ValueFilter,
+} from '@grafana/data/src/transformations/transformers/filterByValue';
+
+import { valueFiltersRegistry, ValueFilterID } from '@grafana/data/src/transformations/valueFilters';
+
+function FilterSelectorRow(props: any) {
```

Comment:
```suggestion
interface RowProps {
  fieldNameOptions: Array<SelectableValue<string>>;
  onDelete: () => void;
  onConfigChange: (config: ValueFilter) => void;
  config: ValueFilter;
  fieldType: FieldType;
}

const FilterSelectorRow: React.FC<RowProps> = props => {
```

I would change it to this so you get proper typings on your child component.